app/services/vot/supported_component_values.rb

```diff
@@ -38,6 +38,12 @@ module SupportedComponentValues
       implied_component_values: [P1],
       requirements: [:biometric_comparison],
     ).freeze
+    Pe = ComponentValue.new(
```

Comment:
I notice other components are incremental, i.e. C1, C2, or Ca, Cb. But `Pe` seems to be using "e" as a shorthand for "enhanced" rather than incrementing. Have we considered this inconsistency?

Reply:
After looking into the background of this, I think `Pe` where "e" stands for "enhanced" is expected, and `Pb` is meant to correspond to "biometric". In other words, no issue from me!